doc/release-notes-16377.md

```diff
@@ -0,0 +1,6 @@
+RPC changes
+-----------
+The `walletcreatefundedpsbt` RPC call will fail with
```

Comment:
The phrasing is vague on whether this is talking about previous or new behavior. Maybe `RPC call will now fail with`?